DQMOffline/L1Trigger/python/L1TriggerDqmOffline_cff.py

```diff
@@ -19,7 +19,9 @@
 
 import DQMServices.Components.DQMEnvironment_cfi
 dqmEnvL1T = DQMServices.Components.DQMEnvironment_cfi.dqmEnv.clone()
-dqmEnvL1T.subSystemFolder = 'L1T'
+dqmEnvL1T.subSystemFolder = 'L1T2016'
```

Comment:
I don't care about the name of this folder myself, but that will most definitely break something.